packages/core/docs/integrations/adyen.md

```diff
@@ -191,4 +191,15 @@ interface AdyenConfigBuilder {
 If the transaction is authorized, the server's controller for `payAndOrder`/`submitAdditionalPaymentDetails` will place an order in Commercetools and apply the `order` object to the response. Thanks to that, we have only one request from the client to both finalize/authorize a payment and make an order.
 
 ## Checkout.com
-Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).
\ No newline at end of file
+Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).
+
+## FAQ
+### Error: NotFound: URI not found: /<project_name>/carts/<cart_id>
+`ctApi.apiHost` property inside your `middleware.config.js` contains wrong path. It should be `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/<project_name>/graphql`
+
+### Error: The type with the key 'ctp-adyen-integration-web-components-payment-type' was not found
+You have to add new types and extension to commercetools as described on these pages:
+- [Extension Module](https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/docs/HowToRun.md#commercetools-project-requirements),
+- [Notification Module](https://github.com/commercetools/commercetools-adyen-integration/blob/master/notification/docs/HowToRun.md#commercetools-project-requirements).   
+
+We've pointed it out in 6th step of the [Adyen's installation guide](./adyen.html#installation).
```

Comment:
```suggestion
For more information, see the 6th step of the [Adyen's installation guide](./adyen.html#installation).
```

IMHO `We've pointed it out in 6th step` sounds like we blame the user for not reading the installation guide carefully :)